.gitignore

```diff
@@ -1,7 +1,7 @@
 # Gradle
 .gradle
 /build/
-/*/build/
+/**/build/
```

Comment:
I think this can now be simplified to `build` which we couldn't use before because we had a `org.junit.gradle` package.